### ISSUE TITLE
百度云盘链接

### ISSUE THREAD

**Issue by yichen115** [BYDcar/BYDRepairManual#1]
链接: https://pan.baidu.com/s/1oW0CQsn2nHPJsGNuU5tfGQ?pwd=4jp7
提取码: 4jp7

**Comment by GM-327** [comment]
Can't acess whithout Chinese phone number. Can you plz upload it in another website and give us the link?

**Comment by yichen115** [comment]
> Can't acess whithout Chinese phone number. Can you plz upload it in another website and give us the link?

Hahaha, I need to buy a VPN service to access GitHub in China. If you are not in China, you should be able to download directly from GitHub?

**Comment by GM-327** [comment]
Yes, but I can't find the Atto 3 (Yuan Plus) repair manual in github.

**Comment by GM-327** [comment]
Can you try this link? www.car60.com and get access to the miniapp in weixin to download the Yuan Plus repair manual?

**Comment by yichen115** [comment]
![image](https://github.com/user-attachments/assets/87da6387-2fc4-4a87-b64a-755997126f3e)
Here it is. If I have time, I will try to download it. There is no obvious download button. I may need time to capture the package or reverse it.

**Comment by GM-327** [comment]
It will be a huge add! Thank you ☺️
谢谢

**Comment by yichen115** [comment]
> It will be a huge add! Thank you ☺️ 谢谢

This is my first time using Google Drive, here is the link: https://drive.google.com/file/d/1JqQ0N20fHafmnKZYyxLTzbPKX1sON9df/view?usp=sharing
